src/server/ApiRoutes/Hold.js

```diff
@@ -287,6 +287,7 @@ function confirmRequestServer(req, res, next) {
  * @return {function}
  */
 function newHoldRequest(req, res) {
+  const loggedIn = User.requireUser(req, res);
```

Comment:
I think you should still `return` something if the user is not logged in. Just to be consistent if express complains.